migrations/20190509094659-add-amountType-column-to-tiers.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+import { map } from 'bluebird';
+
+const colName = 'amountType';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const colParams = {
+      type: Sequelize.ENUM('FLEXIBLE', 'FIXED'),
+    };
+
+    return queryInterface
+      .addColumn('Tiers', colName, colParams)
+      .then(() => {
+        return queryInterface.sequelize.query(`
+          SELECT * FROM "Tiers"
+        `);
+      })
+      .then(results => {
```

Comment:
Like in previous similar PR, it's better to skip the model and do a single SQL query directly.